state/aws/dynamodb/dynamodb.go

```diff
@@ -102,13 +102,32 @@ func (d *StateStore) Set(req *state.SetRequest) error {
 
 // BulkSet performs a bulk set operation
 func (d *StateStore) BulkSet(req []state.SetRequest) error {
+	writeRequests := []*dynamodb.WriteRequest{}
+
 	for _, r := range req {
-		err := d.Set(&r)
-		if err != nil {
-			return err
+		writeRequest := &dynamodb.WriteRequest{
+			PutRequest: &dynamodb.PutRequest{
+				Item: map[string]*dynamodb.AttributeValue{
+					"key": {
+						S: aws.String(r.Key),
+					},
+					"value": {
+						S: aws.String(fmt.Sprintf("%v", r.Value)),
```

Comment:
I made this comment 9 days ago but forgot to publish. Can you validate this for other types? "S:" means always String.

Reply:
@artursouza Thanks for the review 👍 I've updated it, could you please review again?